kie-server-parent/kie-server-services/kie-server-services-drools/pom.xml

```diff
@@ -50,6 +50,36 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
+
+    <!-- drools workbench models -->
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-workbench-models-commons</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-workbench-models-datamodel-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-workbench-models-guided-dtable</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-workbench-models-guided-dtree</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-workbench-models-guided-scorecard</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-workbench-models-guided-template</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-workbench-models-test-scenarios</artifactId>
+    </dependency>
```

Comment:
Very good point Petr, I'll update pom.xml

Dnia 13.11.2015 o godz. 20:42 Petr Široký <notifications@github.com> napisał(a):

> In kie-server-parent/kie-server-services/kie-server-services-drools/pom.xml:
> 
> > +    <dependency>
> > +      <groupId>org.drools</groupId>
> > +      <artifactId>drools-workbench-models-guided-dtree</artifactId>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>org.drools</groupId>
> > +      <artifactId>drools-workbench-models-guided-scorecard</artifactId>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>org.drools</groupId>
> > +      <artifactId>drools-workbench-models-guided-template</artifactId>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>org.drools</groupId>
> > +      <artifactId>drools-workbench-models-test-scenarios</artifactId>
> > +    </dependency>
> It would be great if these would be <scope>runtime</scope> as they are not needed during compile time.
> 
> —
> Reply to this email directly or view it on GitHub.
>